lib/Zonemaster/Backend/RPCAPI.pm

```diff
@@ -67,6 +67,15 @@ sub version_info {
     return \%ver;
 }
 
+$json_schemas{profiles_names} = joi->object->strict;
```

Comment:
I believe "profile_names" would be grammatically better.

Reply:
done :)